models/keyword.go

```diff
@@ -56,7 +56,10 @@ func GetKeywordBy(condition map[string]interface{}) (Keyword, error) {
 	return keyword, nil
 }
 
-func GetKeywordsBy(condition map[string]interface{}) ([]Keyword, error) {
+func GetKeywordsBy(condition interface{}) ([]Keyword, error) {
```

Comment:
Because gorm `Where` function allow both map and string condition. You can pass both following to the function
```go
db.GetDB().Where("user_id = 1").First(&keyword)
```
```go
condition = make(map[string]interface{})
condition["user_id"] = 1
db.GetDB().Where(condition).First(&keyword)
```
But I think your suggestion is better as we can force the type of data to pass in and it is easier to handle.
Fixed in 65afd1cc44daba32fc22b257bfc5bb964b76564b